app/controllers/friend_lists_controller.rb

```diff
@@ -3,8 +3,7 @@ class FriendListsController < ApplicationController
   before_action :check_token_expiration
 
   def index
-    @current_user_friend_lists = current_user.friends
-                                             .select("friend_lists.id AS id, friend_lists.user_id, users.id AS friend_id, users.full_name, users.level")
+    @current_user_friend_lists = FriendList.current_user_friend_lists(current_user)
```

Comment:
If you access the current user globally, then there is no need to pass it as a parameter. You can use it at model level